src/dotnet-scaffolding/dotnet-scaffold-aspnet/ScaffoldSteps/ValidateRazorPagesStep.cs

```diff
@@ -40,6 +45,7 @@ public override async Task<bool> ExecuteAsync(ScaffolderContext context, Cancell
         var codeModifierProperties = new Dictionary<string, string>();
         if (razorPagesSettings is null)
         {
+            _telemetryService.TrackEvent(new ValidateScaffolderTelemetryEvent(nameof(ValidateIdentityStep), context.Scaffolder.DisplayName, result: false));
```

Comment:
Should all of these (other files too) be ValidateIdentityStep? Shouldn't it be ValidateRazorPagesStep in this file, etc?

Reply:
oops yep will fix these